regexes/device/mobiles.yml

```diff
@@ -25899,6 +25953,15 @@ Zopo:
     - regex: '(?:ZOPO_)?(Flash X3|X1i|ZP[0-9]{2,}[^/;)]+)(?: Build|\))'
       model: '$1'
 
+# Redmi (https://www.mi.com/global/phone/#redmi)
+Redmi:
+  regex: 'AFTBTX4(?:[);/ ]|$)'
+  device: 'smartphone'
+  models:
+    - regex: 'AFTBTX4(?:[);/ ]|$)'
+      model: 'Smart Fire TV 4K'
+      device: 'tv'
```

Comment:
Move device to Xiaomi.

Reply:
or transfer Xiaomi Redmi* devices to this brand
starting from the release date of 3 jan. 2019

*before we start, I would like to know your opinion* @sgiehl

Isn't it time for us to create two brands Honor and Redmi and separate the devices?

Reply:
Isn't Redmi still a sub-brand of Xiamoi? In general I wouldn't have problems with splitting it, but at least in Matomo it could cause a bit of problems for people that specifically segment for certain devices or brands...

Reply:
@liviuconcioiu Can you transfer `Redmi Smart Fire TV 4K` to a brand Xiaomi?
It is better to separate the Redmi and Honor devices in another branch.